app/backplane/sensor_module/conf/prj.conf.old

```diff
@@ -67,11 +75,11 @@ CONFIG_TEST_RANDOM_GENERATOR=y # TODO: See if its worth using STM32 HW RNG to re
 CONFIG_SENSOR=y
 CONFIG_SENSOR_ASYNC_API=y
 CONFIG_SENSOR_INFO=y
-CONFIG_TMP116=y
-CONFIG_BMP388=y
-CONFIG_LSM6DSL=y
-CONFIG_ADXL375=y
-CONFIG_MS5611=y
+#CONFIG_TMP116=y
```

Comment:
I assume these changed because the sensor_module_defconfig takes care of enabling them. I think we can probably fully remove these rather than just comment them out.

Reply:
Just removing the whole file. This was marked .old